test/RestSharp.Tests.Serializers.Xml/SampleClasses/twitter.cs

```diff
@@ -105,7 +105,7 @@ public class complexStatus {
     public string in_reply_to_screen_name { get; set; }
 
     // ignore contributors for now
-    [DeserializeAs(Name = "user.following")]
+    [DeserializeAs(Name = "User.following")]
```

Comment:
That's wrong as the message format is camel case.

Reply:
This, was just a thing I found, I just changed the name of the User-Class to starting UpperCase, according to the C# Convention for Classes

Reply:
Yeah, I get it. Have the same urge to fix things sometimes. But, it is better to keep the PR scope to its purpose as things might get break unexpectedly.

Reply:
We can let it unchanged if you want, it's not as important for me